weave-js/src/urls.ts

```diff
@@ -0,0 +1,55 @@
+export const URL_RECENT = 'recent';
```

Comment:
Something that i feel was a mistake in the core app that would be great to avoid:
1. versioning of URI schemas. One idea would be to have a `vX` initial path component to allow use to migrate to completely new schemas in the future.
2. mixing of user-defined variadic path components at the same path index as app-defined hard-coded options. This relates to the asset type comment. I think we hit foot guns when this situation arrises

Reply:
I think it would be good to decide how important "pretty" URLs are to us and at what point we want to commit to backwards compatibility. I'd also like to better understand the longer term vision for this UI & how it will integrate with the main W&B app UI.

Reply:
- Pretty URLs: IMO, it is ok to have longer URLs as long as they are readable if that affords us flexibility in the future. I think we can always invest in a "tiny url" style solution at a W&B level - which i think would benefit a lot of surface areas. Would love to know your thoughts.
- Backwards compat: it very much feels like we should maintain backwards compat moving forward. 
- integrate with the app. I think that the weave.wandb.ai root will continue to serve as a general-purpose weave browser. We may embed it in the app somewhere - or a filtered/truncated version of it. This is definitely still an unknown.

I think i would prefer a `/w/` or `/v0/` prefix to basically allow us to maintain flexibility on these decisions. I suppose the alternative is to just take the risk that we collide with an entity name in the future.

What are your thoughts on #3 above (`multiple variadic paths following eachother.`)?

Reply:
OK, I'll add in a `/v0/`.

Totally agree that it is best to avoid namespace collisions and that this had been the cause of a lot of bugs in the app. Do you want e.g. `/v0/entity/<entity>/project/<project>/assets/<asset_type>/...`?

Reply:
I personally really like that format, but could be convinced that we don't need the intermediate concrete paths for brevity and alignment with W&B. ultimately, OK with whatever you prefer given we have a version now.

Reply:
Discussion moved to internal [Jira doc](https://www.notion.so/wandbai/Weave-UI-URL-Structure-c42375fcfdbc481b9e4821ad99f543cc?pvs=4) and resolved.